static-site/app/groups/[[...groups]]/group-listing-page.tsx

```diff
@@ -55,7 +55,7 @@ export default function GroupListingPage(): React.ReactElement {
   useEffect((): void => {
     async function fetchData(): Promise<void> {
       try {
-        const available = await fetchAndParseJSON<AvailableGroups[]>(
+        const available = await fetchAndParseJSON<AvailableGroups>(
```

Comment:
[question] How can something appease the type checker as an array of X and also just `X`?

Reply:
Gah - what! TS has so many of these gotchas. They really me wonder about the limitations of TS that I am not fully aware of

Anyway, nice work identifying the original mis-typing!

Reply:
I dug a little more. Explanation: string-indexing an array (`AvailableGroups[]["narratives"]` is valid JS which returns an implicit type `any`. This is currently allowed by our TypeScript config which disables `noImplicitAny`. Enabling that (#879) shows an error with the original mis-typing.

Reply:
Switched to `available.narratives` in c6f9ab9508cc37d4a8a1c67dbb08438e568a5d0f